hack/update-generated-completions.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
given we don't have the custom gopath now, can we just reduce it to committing output of `oc completion ${shell}` to a folder and drop `.files_generated`. seems like that's what it will do in the end https://github.com/openshift/origin/tree/master/contrib/completions